### PR TITLE
refactor: renames eth -> rbtc 

### PR DESCRIPTION
**Pull request by jurajpiar**
Renames all references to eth as currency to rbtc

according to [ROLLUP-184](https://rsklabs.atlassian.net/browse/ROLLUP-184)

### CODE REVIEW

**Comment by jurajpiar** [comment]
squash before merging

File: contracts/contracts/ZkSync.sol [hunk 1]
```diff
@@ -206,9 +206,9 @@ contract ZkSync is UpgradeableMaster, Storage, Config, Events, ReentrancyGuard {
         delegateAdditional();
     }
 
-    /// @notice Deposit ETH to Layer 2 - transfer ether from user into contract, validate it, register deposit
+    /// @notice Deposit RBTC to Layer 2 - transfer ether from user into contract, validate it, register deposit
     /// @param _zkSyncAddress The receiver Layer 2 address
-    function depositETH(address _zkSyncAddress) external payable {
+    function depositRBTC(address _zkSyncAddress) external payable {
```
Comment (antomor) [inline contracts/contracts/ZkSync.sol:211]: @rsksmart/rif-rollup-tools surely it's a breaking change for the SDK

**Comment by OnyekaIjeh** [comment]
Could you please summarise the API changes?

File: sdk/zksync.js/src/typechain/ZkSync.d.ts
Comment (antomor) [inline sdk/zksync.js/src/typechain/ZkSync.d.ts:1]: is this file auto-generated?

**Review by antomor — APPROVED** [review]

**Comment by antomor** [comment]
@jurajpiar Should we change the following occurences?
- https://github.com/rsksmart/ri-aggregation/blob/7928ee0c62aea1f8fa007f143cfda29711827f13/core/bin/zksync_forced_exit_requests/src/eth_watch.rs#L589
- https://github.com/rsksmart/ri-aggregation/blob/7928ee0c62aea1f8fa007f143cfda29711827f13/core/bin/zksync_forced_exit_requests/src/eth_watch.rs#L708
- https://github.com/rsksmart/ri-aggregation/blob/7928ee0c62aea1f8fa007f143cfda29711827f13/core/lib/config/src/test_config/mod.rs#L83
   -  And the related `EthConfig`

File: core/tests/loadnext/README.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ OPERATIONS_PER_ACCOUNT
 #
 # Note that we use ERC-20 token since we can't easily mint a lot of ETH on
 # Rinkeby or Ropsten without caring about collecting it back.
+# TODO: ^^^ lose references to non-rootstock networks
```
[the remark this answers is not in the document]
Comment (jurajpiar) [reply core/tests/loadnext/README.md:79]: I think so, but I would rather not do that in this scope.

File: sdk/zksync.js/src/typechain/ZkSync.d.ts
Comment (jurajpiar) [reply sdk/zksync.js/src/typechain/ZkSync.d.ts:1]: yes, I'm not sure why these files are even committed to git, do you?

**Comment by jurajpiar** [comment]
> @jurajpiar Should we change the following occurences?
> 
> * https://github.com/rsksmart/ri-aggregation/blob/7928ee0c62aea1f8fa007f143cfda29711827f13/core/bin/zksync_forced_exit_requests/src/eth_watch.rs#L589
> * https://github.com/rsksmart/ri-aggregation/blob/7928ee0c62aea1f8fa007f143cfda29711827f13/core/bin/zksync_forced_exit_requests/src/eth_watch.rs#L708
> * https://github.com/rsksmart/ri-aggregation/blob/7928ee0c62aea1f8fa007f143cfda29711827f13/core/lib/config/src/test_config/mod.rs#L83
>   
>   * And the related `EthConfig`

So… if at all, it would be in the #75 as the ETH here represents the network rather than the currency.
Also, I wouldn't (I did in the now-closed PR) because it blows up and makes it messy. In fact, I came to the conclusion that it might be best to leave the instances of eth/ETH/Eth that represent the Ethereum methods etc since that's what RSK also uses (to be EVM compatible I suspect).

File: contracts/old_contracts/ZkSync.json
[the remark this answers is not in the document]
Comment (antomor) [reply contracts/old_contracts/ZkSync.json:1]: If it's not used anywhere, we could delete it, but I guess it's used.
If used, I'd just avoid changing it.

File: sdk/zksync.js/src/typechain/ZkSync.d.ts
Comment (antomor) [reply sdk/zksync.js/src/typechain/ZkSync.d.ts:1]: no, I don't by we could investigate on it. BTW my question was: did you generate it using any tools or you just edited the file?

File: core/tests/loadnext/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (antomor) [reply core/tests/loadnext/README.md:79]: I created a [task](https://rsklabs.atlassian.net/browse/ROLLUP-205) for this

File: contracts/old_contracts/ZkSync.json
[the remark this answers is not in the document]
Comment (jurajpiar) [reply contracts/old_contracts/ZkSync.json:1]: I believe they only used to restore data from older zksync instances, which we do not have. However, probably out of scope this one. I would create a task for it tho, as I don't see the need for these on RSK, as no older instances have ever been deployed.

File: sdk/zksync.js/src/typechain/ZkSync.d.ts
Comment (jurajpiar) [reply sdk/zksync.js/src/typechain/ZkSync.d.ts:1]: Just edited the file.

File: contracts/old_contracts/ZkSync.json
[the remark this answers is not in the document]
Comment (antomor) [reply contracts/old_contracts/ZkSync.json:1]: Agreed, so could you please just restore it?

**Review by antomor — CHANGES_REQUESTED** [review]
@jurajpiar if you restore the `old_contracts/ZkSync.sol` file, I'll approve it and then test the changes within the PR #75

File: contracts/contracts/Config.sol [hunk 3]
```diff
@@ -28,7 +28,7 @@ contract Config {
     /// @dev Success flag bytes length
     uint8 internal constant SUCCESS_FLAG_BYTES = 1;
 
-    /// @dev Max amount of tokens registered in the network (excluding ETH, which is hardcoded as tokenId = 0)
```
Comment (franciscotobar) [inline contracts/contracts/Config.sol:31]: there is another `ETH` at line 40. and 65

**Comment by franciscotobar** [comment]
In the Storage.sol contract are some `ETH` words in line 59, 89 and 145.

**Comment by franciscotobar** [comment]
shouldn't we change the `eth>test_config>constant>eth.json` file name.

File: contracts/contracts/ZkSync.sol (hunk 1 above)
Comment (jurajpiar) [reply contracts/contracts/ZkSync.sol:211]: no longer as I understand. @OnyekaIjeh ?

File: contracts/old_contracts/ZkSync.json
[the remark this answers is not in the document]
Comment (jurajpiar) [reply contracts/old_contracts/ZkSync.json:1]: done

**Review by antomor — APPROVED** [review]
LGTM

**Pull request merged**